app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt

```diff
@@ -78,12 +74,14 @@ class OverdueListViewHolder(
     private val phoneCallClickStream: PublishSubject<CallPatientClicked>
 ) : RecyclerView.ViewHolder(itemView) {
 
-  private val patientName by bindView<TextView>(R.id.overdue_patient_name_gender_age)
+  private val patientName by bindView<TextView>(R.id.overdue_patient_name_gender)
   private val patientBP by bindView<TextView>(R.id.overdue_patient_bp)
   private val overdueDays by bindView<TextView>(R.id.overdue_days)
   private val callButton by bindView<ImageButton>(R.id.overdue_patient_call)
   private val separator by bindView<View>(R.id.overdue_separator)
   private val actionsContainer by bindView<LinearLayout>(R.id.overdue_actions_container)
+  private val age by bindView<TextView>(R.id.overdue_patient_age)
+  private val phoneNumber by bindView<TextView>(R.id.overdue_patient_phoneNo)
```

Comment:
Whoops, I just realized that you're not following the naming convention for View fields here. Please fix them. :)